package.json

```diff
@@ -95,6 +95,7 @@
   },
   "dependencies": {
     "babel-runtime": "6.9.2",
-    "vue": "1.0.24"
+    "vue": "1.0.24",
+    "vue-multiselect": "^1.0.1"
```

Comment:
Easy way to rebase package.json, copy from master in the repo, and paste your branch :) And there won't be diff.